doc/index.rst

```diff
@@ -42,43 +44,89 @@ game without learning a complex framework.
     <p><a href="https://vimeo.com/662273899">Quick Arcade Library Introduction Video</a></p>
    </div>
 
-The library was originally designed to be used in an educational setting, but
-it has evolved to be a powerful library for creating games. It is used in
-education, prototyping, and game jams. Hobbyists wanting to create a game
-quickly can benefit from the library's simple API.
-
+Open Source, Libre Assets
+-------------------------
 
-Documentation
--------------
+Arcade is free to use, and that :ref:`includes commercial projects <faq_commercial>`.
+We make sure licensing stays out of the way so you can of focus on really matters: creating.
 
-Arcade provides several forms of documentation depending on the learning style of the user.
-For beginners to programming the Arcade book is a great place to start. For those who prefer
-to learn by examples the Arcade examples are a great resource. We also have various tutorials
-for specific topics. More advanced users may prefer the API reference.
+* :ref:`permissive_mit` for library code
+* :ref:`resources` hand-picked for quality and zero-strings licensing
+* Remixing the :ref:`examples` and other code is highly encouraged
 
-Open Source
------------
+.. pending: post-3.0 cleanup.
 
-Arcade is free and open source. It is released under the MIT License. This means you can use it for free,
-and you can even use it in commercial projects. Copying and adapting content from Arcade in educational
-materials is highly encouraged.
+.. raw:: html
 
+   <br/>
 
 Get Involved
 ------------
 
-Arcade is developed by volunteers.
+Arcade exists because volunteers choose to make it better. If you see
+a bug or a typo, let us know at once the :ref:`community-locations`.
+The following are especially active:
+
+* The `Arcade Discord`_
+* The `Arcade GitHub`_ repo
+
+.. raw:: html
+
+   <br/>
+
+Testing and bug reports are crucial to making the library better,
+especially when the team is working on new features. Arcade's powerful
+OpenGL and `Shadertoy <https://www.shadertoy.com/>`_ features couldn't
+exist without help and contributions from users like you.
+
+We welcome your help. See :ref:`how-to-contribute` for more on how to
+get started.
+
+
+Documentation and Community
+---------------------------
+
+Arcade stays true to its roots in education by provoding
+documentation for different learning styles and experience levels.
+
+* The :ref:`main-page-tutorials` are guided introductions to new skills
+* The :ref:`example-code` help tinkerers get started quickly
```

Comment:
'helps tinkerers'

Reply:
This is one of the cases where the ref title is different than the link target. It resolves to `Examples`.